model/Dockerfile

```diff
@@ -20,6 +20,9 @@ RUN mkdir -p /usr/src/app
 RUN apt update
 RUN apt install -y jq \
     wget \
+    curl \
+    vim \
```

Comment:
Are we really using vim and curl in the container? I guess we should delete these two from the docker image.